src/components/Comment/Comment/styled.ts

```diff
@@ -17,7 +18,7 @@ export const CommentContent = styled.div`
 
 export const CommentAuthor = styled.p`
   display: inline-block;
-  margin-right: 5px;
+  margin: 10px 10px 0 0;
```

Comment:
Just realized that this doesn't use typography spacing. Should probably change to that, but not a huge issue if not